src/renderer/Camera.ts

```diff
@@ -0,0 +1,99 @@
+import { mat4, quat, vec3 } from 'gl-matrix';
+
+/**
+ * Represents the orientation of the camera in a scene, defined by a position for the camera and a
+ * target for the camera to look at
```

Comment:
This is a huge nit, but do you think you could use periods at the end of your comments to be consistent with the commenting I have in #11?